### DISCUSSION TITLE
Tabs

### DISCUSSION THREAD

**Discussion by philipbaginski** [laravel/nova-issues#4044] [Ideas & Feature Requests]
There is no single website/app in nowadays without tabs. If you have few related resources, that's Okay. But if you have many of them and want to show on Detail page, scrolling sounds like very important skill.

Please, take a look at https://github.com/eminiarts/nova-tabs tabs and instead of Panel implement tabs 😃

**Comment by vv-ac** [comment]
I second this, tabs would boost the detail pages of resources to unknown limits. Especially when it comes to more complex resources incl. mutliple relationsships, it's hard to have all of the information displayed in "one tab".

**Comment by wdelfuego** [comment]
That package from Emini Arts you link looks very nice. Looks like they are currently [working on Nova 4 compatibility](https://github.com/eminiarts/nova-tabs/pull/185).

**Reply by philipbaginski** [reply]
> That package from Emini Arts you link looks very nice. Looks like they are currently [working on Nova 4 compatibility](https://github.com/eminiarts/nova-tabs/pull/185).

It works perfect with Nova 3. I have no idea why Nova 4 is out of tabs like that 😜

**Reply by wdelfuego** [reply]
Tabs were always a third party feature, it makes sense that external providers need to do some work to adapt to a new major version. I'm sure it's a matter of time :).

**Reply by philipbaginski** [reply]
> Tabs were always a third party feature, it makes sense that external providers need to do some work to adapt to a new major version. I'm sure it's a matter of time :).

Yes. But it's quite important part of any app. Like Panel in Nova - you can use it, but you don't have to. Still having as an option. The same with tabs :-)

**Reply by marcflp** [reply]
> > Tabs were always a third party feature, it makes sense that external providers need to do some work to adapt to a new major version. I'm sure it's a matter of time :).
> 
> Yes. But it's quite important part of any app. Like Panel in Nova - you can use it, but you don't have to. Still having as an option. The same with tabs :-)

https://github.com/marcfilipan/nova-tabs you can point composer to here as the PR isn't accepted yet. Works with Nova 4.

**Reply by philipbaginski** [reply]
I have seen it. I have a problem to install it. Using composer command, but getting error and also I have no Trait folder in root package folder. Any idea?

**Comment by vladcazacu** [comment]
This would be amazing. With complex resources on the vanilla UI you end up with very long detail pages and forms that are just unusable. I've also been using the tabs plugin which is now supporting v4, but it still has some issues that are blocking me from upgrading. Would be great if Nova provided this out of the box and supported managing more complex resources.

**Comment by ghost** [comment]
This functionality would still be greatly appreciated.

**Comment by miki131** [comment]
Does anyone knows some alternatives for https://github.com/eminiarts/nova-tabs package?

**Reply by msucevan** [reply]
Is working pretty good, for now, no? I know that you mean that is not maintained and this will lead a lot of troubles with Nova 5

**Reply by anditsung** [reply]
there is new tab field on nova 5

**Reply by miki131** [reply]
Where you got such info?

**Reply by anditsung** [reply]
https://github.com/laravel/nova-docs/pull/582/commits/ead6a4312ef36947aceb76617b6d62a803614467

**Reply by miki131** [reply]
Thanks. Any info about when it will be released?

**Reply by anditsung** [reply]
last info i got is when it done with docs and code reviews, so no ETA

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
